code-challenges.test.js

```diff
@@ -38,8 +65,32 @@ const votes1 = { upVotes: 13, downVotes: 2 }
 const votes2 = { upVotes: 2, downVotes: 33 }
 // Expected output: -31
 
+// JEST Test
+// describe finalTally
+// it takes in the two values of votes1, determines which is the larger value, and performs subtraction to get the finalTally
+// expected output: 11
+// it takes in the values of votes 2, compares the two values, and perform addition to get the finalTally
+// expected output: -31
```

Comment:
Look at the prompt again. It does not asks for the larger number. It just wants the upVotes - downVotes.
However, you will need to consider how to obtain the value from an object. 
Checkout dot notation:
https://github.com/learn-academy-2022-hotel/Syllabus/blob/main/javascript/objects.md